Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.6.9-alpine3.10
+LABEL maintainer="Hypothes.is Project and contributors"
+
+# Install nginx & supervisor
+RUN apk add --no-cache nginx gettext supervisor \
+    && mv /usr/bin/envsubst /usr/local/bin/envsubst \
+    && mkdir -p /run/nginx
```

Comment:
?

Reply:
Apparently this directory doesn't exist on the latest alpine containers so we have to add it manually. Alternatively we can specify an alternative location in the nginx.conf which we could do in the log run instead. See [answer here](https://superuser.com/questions/1306656/cant-run-nginx-in-alpine-linux-docker).

Reply:
I've moved the path to file into the nginx.conf.